content/es/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,75 @@
+---
+reviewers:
+  - edithturn
+  - raelga
+  - electrocucaracha
+title: Capacidad de Almacenamiento
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+La capacidad de almacenamiento es limitada y puede variar según el nodo en el que un Pod se ejecuta: es posible que no todos los nodos puedan acceder al almacenamiento conectado a la red o que, para empezar, el almacenamiento sea local en un nodo.
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.21" state="beta" >}}
+
+Esta página describe cómo Kubernetes realiza un seguimiento de la capacidad de almacenamiento y cómo el planificador usa esa información para programar Pods en nodos que tienen acceso a suficiente capacidad de almacenamiento para los volúmenes restantes que faltan. Sin el seguimiento de la capacidad de almacenamiento, el programador puede elegir un nodo que no tenga suficiente capacidad para aprovisionar un volumen y se necesitarán varios reintentos de programación.
```

Comment:
La frase "varios reintentos de programación" tiene sabor a poco a mi gusto, creo que sería mejor algo como "varios intentos de despliege/planificacion" ?

Reply:
Entendido! Planificador suena mejor definitivamente :)

Reply:
Changed it!  👍🏼 💯